src/Engine/ProtoCore/Reflection/MemberInfoReflectionExtensions.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Autodesk.DesignScript.Runtime;
+
+namespace ProtoCore.Reflection
```

Comment:
I think a more appropriate namespace would be `ProtoFFI.Reflection` but I'm not sure - my thought is, we're not performing reflection on the protocore (Dynamo VM) types, we're just using reflection to inspect CLR modules)